org.eclipse.dartboard/assets/messages.properties

```diff
@@ -17,3 +17,20 @@ ProjectNature_DeletePubspec_Message=Do you also want to delete the pubspec.yaml
 Launch_NoConfigurationFound_Title=No launch configuration available
 Launch_NoConfigurationFound_Body=No launch configuration was found for this project. \
 Please create one first.
+
+NewProject_windowTitle = New Dart Project
+NewProject_title = Dart Project
+NewProject_description = Create a new Dart project.
+NewProject_errorMessage = Creation Problems
+NewProject_internalError = Internal error: {0}
+NewProject_caseVariantExistsError = The underlying file system is case insensitive. There is an existing project or directory that conflicts with ''{0}''.
+
+NewProject_group_label = Dart SDK
+NewProject_sdk_version_label = Dart Version:
+
+NewFile_windowTitle = New Dart File
+NewFile_title = Dart File
+NewFile_description = Create a new Dart file.
+NewFile_creating = Creating file {0}
+NewFile_container_doesnot_exist = Container {0} does not exist.
+NewFile_opening_file = Opening file for editing...
```

Comment:
Please follow the existing style in this file and remove the surrounding whitespace around the = sign.

Please also use PascalCase for the parts after the first underscore (_) Like: NewFile_OpeningFile